src/main/models/caseSubscription.ts

```diff
@@ -1,11 +1,19 @@
 export interface CaseSubscription {
```

Comment:
My understanding is subscriptions aren't specific to a hearing, and are just general search criteria based on a caseName, caseNumber and urn that is then stored as a subscription, therefore I'm not sure we need hearing ID, date / judge / platform / court ID. Would be good to discuss, as I could be wrong on this.

Reply:
Following a discussion, I have raised the investigation into the granularity of subscriptions into a separate ticket: https://tools.hmcts.net/jira/browse/PUB-954.

Reply:
this model is a hearing however, just duplicated for some reason with another name

Reply:
Resolved as this has been removed and hearing model is used instead